tasks/delete_app_objects_kind.yml

```diff
@@ -1,30 +1,31 @@
 # Delete a kind of objects for the targeted deployment_stamp
-- name: Lookup {{ kind }} to delete with deployment_stamp {{ targeted_deployment_stamp }} for app {{ app.name }}
+- name: Lookup {{ resource_to_delete.kind }} to delete with deployment_stamp {{ targeted_deployment_stamp }} for app {{ app.name }}
```

Comment:
because by default he value is `v1` see https://docs.ansible.com/ansible/latest/plugins/lookup/k8s.html

Reply:
wow that sucks. Have you tried setting it to null?

Reply:
No, do you think it's better than what I made ?

Reply:
well it would be simpler and would avoid looping on all versions when you get several?

Reply:
I don't think we loop on all versions, we use only the one defined in the hashmap

Reply:
Ok up to you but it is also uglier that you have to pass a whole object to define what you want to delete... so if the version could be ignored it would be better imo. If null doesn't work then ok lah.

Reply:
I tried and no resources are deleted, the `delete_app` fails with a timeout